session_handler.go

```diff
@@ -47,6 +51,34 @@ func NewSessionHandler(cfg SessionHandlerConfig) http.Handler {
 	}
 }
 
+// getCallerName is adapted from the runtime pkg examples for the Callers fn
```

Comment:
might be helpful to have a link for future reference

Reply:
https://github.com/Clever/mgohttp/pull/7/commits/81afca6ad3c82a29918f9f03d2bd6751db9b00cf